napari/_qt/qt_range_slider_popup.py

```diff
@@ -72,7 +72,7 @@ def _on_text_changed(self, text):
     def update_position(self):
         """Update slider position."""
         x = self.get_pos() - self.width() / 2
-        y = self.slider.handle_radius + 6
+        y = self.slider.handle_radius + 12
```

Comment:
@tlambert03 I approved but now I find myself wondering whether this number is dependent on whether you have a HiDPI screen...

Reply:
possibly.  But if so, then it falls into a big category of similar things that need fixing in some High DPI PR.  My gut feeling as that, with the HIGH_DPI env variable set... this works fine (I say that because I use a high DPI monitor at home, and when I boot into linux, it had been working with the previous number, which was similarly hard-coded)